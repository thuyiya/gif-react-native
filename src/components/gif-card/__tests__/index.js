/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { GifCard } from "_components";

describe("GifCard", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <GifCard thumbnailSource={""} source={""} title={""} url={""} />
    );
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
