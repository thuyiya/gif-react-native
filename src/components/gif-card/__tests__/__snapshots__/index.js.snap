// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifCard renders correctly react-test-renderer 1`] = `
<React.Fragment>
  <View
    style={
      Object {
        "height": 718,
        "width": 718,
      }
    }
  >
    <Skeleton />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginTop": 8,
      }
    }
  >
    <View
      accessible={true}
      style={
        Object {
          "width": 654,
        }
      }
    >
      <Text
        accessibilityAutoFocus={true}
        accessibilityRole="header"
        style={
          Object {
            "color": "#000000",
            "marginTop": 16,
          }
        }
      >
        
      </Text>
      <Text
        style={
          Object {
            "marginTop": 8,
          }
        }
      >
        
      </Text>
    </View>
    <AgeBadge
      fontSize={22}
      style={
        Object {
          "borderRadius": 36,
          "height": 72,
          "width": 72,
        }
      }
      text="16+"
    />
  </View>
</React.Fragment>
`;
